docs/config.md

```diff
@@ -66,6 +66,11 @@ Supported keys include:
 | Key | Description | Supported Values | Default |
 | :--- | :--- | :--- | :--- |
 | `signers.kms.kmsref` | The URI reference to a KMS service to use in `KMS` signers. | `gcpkms://projects/[PROJECT]/locations/[LOCATION]>/keyRings/[KEYRING]/cryptoKeys/[KEY]`| |
+| `signers.kms.auth.address` | URI of KMS server (e.g. the value of `VAULT_TOKEN`) | |
```

Comment:
WDYT of marking this as an experimental feature for now until we can get some usage/feedback. That'll make it easier to make changes to config later if we need to!

We can move this documentation into the [experimental features section](https://github.com/tektoncd/chains/blob/74ac70b0ccf9cb6a45e36fe0e5bbc7075cb23ab9/docs/config.md#experimental-features-configuration)